ipn/localapi/localapi.go

```diff
@@ -718,6 +719,34 @@ func (h *Handler) serveStatus(w http.ResponseWriter, r *http.Request) {
 	e.Encode(st)
 }
 
+func (h *Handler) servePeerStatus(w http.ResponseWriter, r *http.Request) {
```

Comment:
peerStatus seems like a bad name here, lets be super specific `serveDebugPeerEndpointChanges`
same in the url

Reply:
Done